### DISCUSSION TITLE
Data1D/2D refactoring ensemble programming

### DISCUSSION THREAD

**Discussion by lucas-wilkins** [SasView/sasview#2248] [Code Camp Topics]
Paired/ensemble programming for refactoring that data structures in sasdata.

@krzywon @rozyczko Do you guys use either pyCharm or VSCode?

**Comment by krzywon** [comment]
I'm team PyCharm and have used the [Code With Me](https://www.jetbrains.com/code-with-me/) features a number of times.

**Comment by rozyczko** [comment]
VS/VSCode FTW. Live Share as a pair programming tool.
